src-frontend/tests/unit/components/procedure-card-test.js

```diff
@@ -0,0 +1,18 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
Having this as a separate issue to deal with all the tests generated. Can we configure ember generators to make it 4 spaces?